src/views/name.njk

```diff
@@ -46,21 +46,13 @@
 
       </div>
 
-      <details class="govuk-details">
-        <summary class="govuk-details__summary">
-          <span class="govuk-details__summary-text">
-            {{ i18n.namePage.checkName }}
-          </span>
-        </summary>
-        <div class="govuk-details__text">
-          <p>
-            {{ i18n.namePage.checkNameExistingLP | safe }}
-          </p>
-          <p>
-            {{ i18n.namePage.checkNameExistingTrademark | safe }}
-          </p>
-        </div>
-      </details>
+      {{ govukInsetText({
+        html: 
+        '<p>You must check the name is not the same as an existing partnership or company on the <a href="https://find-and-update.company-information.service.gov.uk/company-name-availability">Companies House register (opens in a new tab)</a>. If it is, we’ll reject your application.</p>
```

Comment:
we should add the text to the translations.json files

Reply:
Introduced localisation